tools/pepsi/src/upload.rs

```diff
@@ -38,6 +38,40 @@ pub struct Arguments {
     /// The NAOs to upload to e.g. 20w or 10.1.24.22
     #[arg(required = true)]
     pub naos: Vec<NaoAddress>,
+    #[arg(long)]
+    pub skip_check_os: bool,
+}
+
+pub async fn check_os(nao: &Nao) -> bool {
+    let os_stable = "5.1.1";
```

Comment:
The latest OS version is `5.2.0`, [released](https://github.com/HULKs/meta-hulks/releases) two days ago.

Reply:
Corrected